src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/format/EncodingService.kt

```diff
@@ -1,28 +1,24 @@
-package uk.gov.justice.digital.hmpps.personrecord.message.processors.probation
+package uk.gov.justice.digital.hmpps.personrecord.service.format
 
 import kotlinx.coroutines.runBlocking
 import org.slf4j.LoggerFactory
-import org.springframework.beans.factory.annotation.Autowired
 import org.springframework.beans.factory.annotation.Value
-import org.springframework.stereotype.Component
+import org.springframework.stereotype.Service
 import uk.gov.justice.digital.hmpps.personrecord.client.CorePersonRecordAndDeliusClient
+import uk.gov.justice.digital.hmpps.personrecord.client.PrisonerSearchClient
 import uk.gov.justice.digital.hmpps.personrecord.client.model.offender.ProbationCase
+import uk.gov.justice.digital.hmpps.personrecord.client.model.prisoner.Prisoner
 import uk.gov.justice.digital.hmpps.personrecord.service.RetryExecutor
 
-@Component
-class BaseProbationEventProcessor {
-
-  @Autowired
-  private lateinit var corePersonRecordAndDeliusClient: CorePersonRecordAndDeliusClient
+@Service
+class EncodingService(
```

Comment:
Better name for this welcomed!

Reply:
I think it's hard to name because it does 2 things. It would be something like a `PrisonerSearchService` or a `ProbationSearchService`
What do we gain by having these 2 methods in this class?